Makefile

```diff
@@ -15,12 +15,17 @@ VERSION := $(shell grep -v ^\# $(VERSION_FILE))
 COMMIT_NO := $(shell git rev-parse HEAD 2> /dev/null || true)
 COMMIT := $(if $(shell git status --porcelain --untracked-files=no),${COMMIT_NO}-dirty,${COMMIT_NO})
 VERSION_COMMIT := $(if $(COMMIT),$(VERSION)-$(COMMIT),$(VERSION))
+ARCH := $(shell uname -m)
 
 $(TARGET): $(SOURCES)
 	go build -o $@ -ldflags "-X main.version=$(VERSION_COMMIT)"
 
 test:
+ifeq ($(shell [ ${ARCH} = "x86_64" -o ${ARCH} = "amd64" ] && echo true ),true)
```

Comment:
Hi @basheerk - thanks for raising this PR! There is actually an open PR to switch the proxy to using the central `go test` script (#33) which is designed to be used for all the kata code repositories.

I suggest that you close this PR and raise a PR to add `-race` conditionally to `go_test_flags=` in 
https://github.com/kata-containers/tests/blob/master/.ci/go-test.sh. One advantage of doing this being that you get to add this arch support to all the kata code repos in one go ;)

Reply:
@jodh-intel Thanks, as suggested  I will raise PR adding the changes in tests repo